apps/src/templates/teacherHomepage/StudentHomepage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import RecentCoursesCollapsible from './RecentCoursesCollapsible';
+import CollapsibleSection from './CollapsibleSection';
+import GradientNavCard from './GradientNavCard';
+import shapes from './shapes';
+import i18n from "@cdo/locale";
+
+const StudentHomepage = React.createClass({
+  propTypes: {
+    courses: shapes.courses
+  },
+
+  componentDidMount() {
+    $('#user_hero').appendTo(this.refs.userHero).show();
```

Comment:
I have a strong opinion that this hack of moving things into the React DOM on mount should be avoided if at all possible. In the other case, where we have stuff we need to be in the middle of our DOM content, I think it's unfortunate but justified. In this case, I think it's much cleaner to just have the hero live outside of React.